docs/markdown/Writing Plugins/common-plugin-tasks/plugin-helpers.md

```diff
@@ -1,9 +1,10 @@
 ---
 title: "Plugin helpers"
-slug: "plugins-helpers"
+slug: "plugin-helpers"
 excerpt: "Helpers which make writing plugins easier."
-hidden: false
+hidden: true
```

Comment:
hidden because this relates to the old way of generating tool lockfiles, which we'd like people not to use.